src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/AvailabilityDataType.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GraphQL.DataLoader;
+using GraphQL.Resolvers;
+using GraphQL.Types;
+using MediatR;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.ExperienceApiModule.Core.Helpers;
+using VirtoCommerce.InventoryModule.Core.Model;
+using VirtoCommerce.XDigitalCatalog.Queries;
+
+namespace VirtoCommerce.XDigitalCatalog.Schemas
+{
+    public class AvailabilityDataType : ObjectGraphType
```

Comment:
Make this generic or pass arguments to get values from mediator. If this not possible clear constructor.